Dockerfile

```diff
@@ -643,6 +643,11 @@ COPY --link --from=containerutil /build/ /
 COPY --link --from=vpnkit        /       /
 COPY --link --from=build         /build  /
 
+# usage:
+# > docker buildx bake bin-image
+FROM all AS bin-image
+COPY --link contrib/init /contrib/init
```

Comment:
Wondering if we should limit it to the  systemd units only if those are the only ones we need.

The other init scripts are not well-maintained, and we don't really use them / test them 🤔

Reply:
Yeah I'm fine with it